apps/commuter_rail_boarding/test/boarding_status_test.exs

```diff
@@ -94,11 +94,32 @@ defmodule BoardingStatusTest do
       assert from_firebase(result) == from_firebase(original)
     end
 
+    test "assigns a stop ID based on the stop name" do
+      original = List.first(@results)
+
+      no_track = %{
+        original
+        | "gtfs_stop_name" => "North Station",
+          "track" => ""
+      }
+
+      with_track = %{
+        original
+        | "gtfs_stop_name" => "North Station",
+          "track" => "1"
+      }
```

Comment:
I was confused why the formatter did this, but it turns out we have a `.formatter.exs` in this project that overrides the default line length (it's 80 instead of 98). Should we change this to better align with our other projects?

Reply:
That's fine, but ideally in a separate PR.